### PR TITLE
Add contributor's guide

### PR DESCRIPTION
**Pull request by jromero**
This PR adds a contributor's guide in hope of answering a few recurring questions as well as providing a starting point for on-boarding contributors.

NOTE: The best way to review this is probably to look at the [branch on GH](https://github.com/buildpacks/community/blob/contributors-guide/contributors/guide.md).

### CODE REVIEW

File: contributors/guide.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+3. Ongoing efforts aren't disrupted.
+
+To learn more about how you can help triage issues, check out our dedicated [triage][triage] page.
+
+## FAQs
+
+- How do I become a project contributor?
+    
+    The process for becoming a [project contributor](#project-contributor) simply requires that you actively [contribute](#contributions) to the project. There is no set amount of contributions that are expected, but the more involved you are in the project the more votes that would be tipped in your favor. 
+
```
Comment (dfreilich) [inline contributors/guide.md:114]:
```suggestion
    
- How do I become a project maintainer?

```

File: contributors/guide.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+To learn more about how you can help triage issues, check out our dedicated [triage][triage] page.
+
+## FAQs
+
+- How do I become a project contributor?
+    
+    The process for becoming a [project contributor](#project-contributor) simply requires that you actively [contribute](#contributions) to the project. There is no set amount of contributions that are expected, but the more involved you are in the project the more votes that would be tipped in your favor. 
+    
+    Once you've got a decent amount of contributions, you may self-nominate, or be nominated by another team member. The maintainers of said relevant sub-teams will then vote for your election into becoming a project maintainer.
```
Comment (dfreilich) [inline contributors/guide.md:115]:
```suggestion
    Once you've got a decent amount of contributions, you may self-nominate to be a maintainer, or be nominated by another team member. The maintainers of said relevant sub-teams will then vote for your election into becoming a project maintainer.
```

File: contributors/triage.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Triage
+
+Triaging is the act of managing issues, determining the appropriate response, and following up.
```
Comment (dfreilich) [inline contributors/triage.md:3]:
```suggestion
Triaging is the act of managing issues, determining the appropriate response, and following up with the original poster to ensure the problem was solved.
```

File: contributors/triage.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+### Actionable Issues
+
+An actionable issue is one that can be worked on such that the following is true:
```
Comment (dfreilich) [inline contributors/triage.md:7]:
```suggestion
An actionable issue is one that can be worked on, such that the following is true:
```

File: contributors/triage.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+- Definition of done - that may be acceptance criteria or clear expectations.
```
Comment (dfreilich) [inline contributors/triage.md:9]:
```suggestion
- A definition of done is provided - whether that is formal acceptance criteria or clear expectations.
```

File: contributors/triage.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+- Appropriate labels
```
Comment (dfreilich) [inline contributors/triage.md:10]:
```suggestion
- Appropriate label(s) is present on the issue
```

File: contributors/triage.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    - `type/*`
+    - `size/*` (if possible)
+    - `status/ready`
+    - `good first issue` (if applicable)
+
+### Actions
+
+The following is a flow to assist contributors in assigning the appropriate labels in script format.
```
Comment (dfreilich) [inline contributors/triage.md:18]:
```suggestion
The following is a flow to assist contributors in assigning the appropriate labels to an issue.
```

File: contributors/triage.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+        - Remove label `status/triage`
+        - Add label `status/needs-information`
+        - Request further information from issue author
+        - If no further activity for 10 days, close
+    - Yes
+        - Could this be worked on right now?
+            - Yes
+                - Remove label `status/triage`
+                - Add label `status/ready`
+                - Add appropriate `size/*` (if possible) and `type/*`
```
Comment (dfreilich) [inline contributors/triage.md:59]:
```suggestion
                - Add appropriate `size/*` (if possible), `type/*`, and `good-first-issue` if it is relatively straightforward
```

File: contributors/guide.md (hunk 1 above)
Comment (jromero) [reply contributors/guide.md:114]: I misspoke in the answer to the initial question, it was intended to detail Project Contributor.

File: contributors/guide.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+This is anyone that interacts with this project, creates an issue, opens a PR, adds a comment, etc. You are all in one way
+contributing to a better project.
+
+#### Project Contributor
+
+This group of contributors are recognized as providing sizeable contributions to the project.
+
+Additional Responsibilities:
+* Maintain active participation in project.
```
[the remark this answers is not in the document]
Comment (jromero) [reply contributors/guide.md:31]: I'm not sure that I would consider this a responsibility. It's definitely a nice-to-have and keeps the contributor more up-to-date with upcoming changes but I wouldn't consider it something that's mandatory.

**Pull request merged**